docs/datamodel/computables.rst

```diff
@@ -31,3 +37,17 @@ last name:
      }
 
 Computables are also often used in :ref:`views <ref_datamodel_views>`.
+For example, using the ``User`` from the above example, a ``UserView``
+can be defined with a ``lastname_first`` computable which lists the
+full name in the format which is often used in formal alphabetized
+lists:
+
+.. code-block:: sdl
+
+    view UserView := User {
+        property lastname_first :=
+            (__source__.lastname ++ ', ' ++
+             __source__.firstname);
+     }
+
+Computables are used in :ref:`shapes <ref_eql_expr_shapes>`, too.
```

Comment:
are used -> can be used?  I think it'd be a good idea to illustrate that here.

Reply:
Fixed the wording and added a SELECT shape example.